config/exec_resource.go

```diff
@@ -111,15 +111,22 @@ func (e *execResource) validate(ctx context.Context) (*ManagedResources, error)
 		return nil, err
 	}
 
-	e.enforcePath, err = e.download(ctx, e.GetEnforce())
-	if err != nil {
-		return nil, err
+	// Assume lack of Enforce means policy is in VALIDATE mode.
```

Comment:
Instead of assuming, can the mode be passed so that we can raise an error if the pre-condition is not satisfied?

Reply:
Not with how this is setup, resources know nothing about policies and there is no per resource customization options.

Reply:
I see. can you add a todo for it for reference.

Reply:
This isn't a TODO I really want to fix as it is just not possible with the resource setup. I don't think this is that big of an issue though seeing as the API should prevent this anyway, also there will be an error at enforce execution time so the end result would be the same with the agent returning an error for this resource.